libraries/standard/mqtt/src/mqtt.c

```diff
@@ -120,34 +153,50 @@ MQTTStatus_t MQTT_Connect( MQTTContext_t * const pContext,
         bytesSent = sendPacket( pContext,
                                 pContext->networkBuffer.pBuffer,
                                 packetSize );
+        IotLogDebugWithArgs( "Sent %d bytes of CONNECT packet. ",
```

Comment:
This log seems duplicated in the else below.

Reply:
Thanks. Fixed it